### ISSUE TITLE
Software Manager Icon not opening into window

### ISSUE THREAD

**Issue by Ringland69** [linuxmint/mintinstall#433]
Installed Linux 22 a couple of days ago as a complete wipe and install.
All works fine except that the Software Manager Icon when selected does not result in window opening on desktop. Instead I see a very quick ghosted image of a growing sized Software Manager window which shoot out of sight to the right of the screen. I thought it may be going to my 2nd Workspace but it's not there.
I have done Sudo Apt Install Mintinstall.
I have done sudo apt-get update && sudo apt-get dist-upgrade.
I have closed down and rebooted.
Still not opening.
Any further suggestions would be gratefully appreciated.
Cheers

**Comment by mtwebster** [comment]
Please paste the output (if any) from:
```
MINTINSTALL_DEBUG=1 DEBUG=1 mintinstall
```

**Comment by Ringland69** [comment]
Hi Mark. Result is as follows:
andrew@andrew-RC520:~$ MINTINSTALL_DEBUG=1 DEBUG=1 mintinstall
MintInstall: low resolution detected (768px height), limiting window height.
MintInstall: Detected system architecture: 'x86_64'
Application.add_categories took 7.733 ms

IMPORTANT: Because of the low resolution detected comment in the output I played around in Display Settings change resolution. Applied then changed back to 
1366x768. Still the same problem. I then changed setting to Mirror but cancelled it's acceptance. Strangely the Software Manager window is now opening as normal.
So there must be a bug in there somewhere but have somehow fixed it but only time will tell if a permanent fix.
Cheers
Andrew
